web/src/components/LoginEmailCheck.tsx

```diff
@@ -0,0 +1,98 @@
+import { Content } from "@/components/Content";
+import { GenericTextField } from "@/components/GenericTextField";
+import { WithErrorBar } from "@/components/WithErrorBar";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { postUserEmailCheck } from "@/lib/api-client/apiClient";
+import { triggerSignIn } from "@/lib/auth/sessionHelper";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import analytics from "@/lib/utils/analytics";
+import { validateEmail } from "@/lib/utils/helpers";
+import { InputLabel } from "@mui/material";
+
+import { useState, type ReactElement } from "react";
+
+export const LoginEmailCheck = (): ReactElement => {
+  const [email, setEmail] = useState<string>("");
+  const [emailError, setEmailError] = useState<string>("");
+
+  const handleSubmit = (email: string): void => {
+    const isValidEmail = validateEmail(email);
+
+    if (isValidEmail) {
+      checkEmailExists(email);
+    } else {
+      setEmailError(Config.checkAccountEmailPage.invalidEmailError);
+    }
+  };
+
+  const checkEmailExists = async (email: string): Promise<void> => {
+    try {
+      const response = await postUserEmailCheck(email);
+      if (response.found) {
+        setEmailError("");
+        triggerSignIn();
+        analytics.event.check_account_next_button.submit.go_to_myNJ_login();
+      }
+    } catch (error) {
+      if (error === 404) {
+        setEmailError(Config.checkAccountEmailPage.emailNotFoundError);
+      } else if (error === 500) {
+        setEmailError(Config.checkAccountEmailPage.serviceNotAvailableError);
+      } else {
+        setEmailError(Config.checkAccountEmailPage.defaultErrorMessage);
+      }
+    }
+  };
+
+  const { Config } = useConfig();
+
+  const handleTextInputChange = (value: string): void => {
+    setEmail(value);
+  };
+
+  return (
+    <div className="email-check-card padding-5 desktop:margin-x-2 radius-md">
+      <Heading level={2} className="text-normal">
+        {Config.checkAccountEmailPage.header}
+      </Heading>
+
+      <WithErrorBar hasError={Boolean(emailError)} type="ALWAYS">
+        <InputLabel htmlFor="email">Email</InputLabel>
+        <GenericTextField
+          inputWidth="default"
+          fieldName="email"
+          value={email}
+          handleChange={handleTextInputChange}
+          error={Boolean(emailError)}
+          validationText={emailError}
+        ></GenericTextField>
```

Comment:
pebble - this can probably be end tagged
```
<GenericTextField
          inputWidth="default"
          fieldName="email"
          value={email}
          handleChange={handleTextInputChange}
          error={Boolean(emailError)}
          validationText={emailError}
        />
```
instead.

Reply:
good catch, I'm surprised prettier doesn't do that one automatically